pom.xml

```diff
@@ -108,7 +108,6 @@
         <csv.version>1.4</csv.version>
         <commons.compress.version>1.21</commons.compress.version>
         <lang3.version>3.3.1</lang3.version>
-        <guava.version>30.1.1-jre</guava.version>
```

Comment:
Where is `guava.version` set if we remove it from here ?

Reply:
With this change I unpin guava so that the version from common is used. https://github.com/confluentinc/common/blob/32655ad5c2accb824968d8ddf94d919d2716313a/pom.xml#L64